chromium/rules.js

```diff
@@ -273,9 +273,9 @@ RuleSets.prototype = {
     }
 
     var cookierules = ruletag["securecookie"];
-    if (cookierules != null) {
+    if (cookierules) {
```

Comment:
In these cases, can you please explain why the variable will never be falsey but not `undefined` or `null`?

Reply:
It will always be either an array (always truthy), or `null` or `undefined`.